static/src/javascripts/projects/commercial/modules/messenger/background.ts

```diff
@@ -88,6 +88,7 @@ const setCtaURL = (
 	ctaURLAnchor.appendChild(backgroundParent);
 	ctaURLAnchor.style.width = '100%';
 	ctaURLAnchor.style.height = '100%';
+	ctaURLAnchor.style.display = 'block';
```

Comment:
Minor: Was this previously `inline-block`?

```suggestion
	ctaURLAnchor.style.display = 'inline-block';
```
https://github.com/guardian/frontend/pull/25407/files#diff-2864ceba9c391a73da143043132416e22f2e0f5f0323b16d72450d47e64cab29L151

Reply:
Oh thanks!